.changeset/smart-buttons-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": patch
+---
+
+Update Slider component implementation to have separate `Slider` and `RangeSlider` components, with improved accessibility and test coverage.
```

Comment:
can we add a basic example in code, so when reading the changelog I get  an understanding in code of what the API looks like, see other changelogs for when we added new components.